src/factories/request.js

```diff
@@ -93,14 +93,18 @@ class RequestFactory {
           'X-MOLTIN-SDK-VERSION': config.sdk.version
         }
 
-        if (config.application) {
-          headers['X-MOLTIN-APPLICATION'] = config.application
-        }
+        headers['X-MOLTIN-APPLICATION'] = config.application
+          ? config.application
+          : 'epcc sdk v2'
```

Comment:
@RyanKirby89 I would drop `v2` so it doesn't cause any confusion with the actual version number header 👀

Reply:
done